pkg/flags/prow.go

```diff
@@ -0,0 +1,30 @@
+package flags
+
+import (
+	"net/url"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/pflag"
+)
+
+type ProwFlags struct {
+	URL string
+}
+
+func NewProwFlags() *ProwFlags {
+	return &ProwFlags{
+		URL: "https://prow.ci.openshift.org",
+	}
+}
+
+func (f *ProwFlags) BindFlags(fs *pflag.FlagSet) {
+	fs.StringVar(&f.URL, "prow-url", f.URL, "URL for Prow")
+}
+
+func (f *ProwFlags) Validate() error {
+	_, err := url.ParseRequestURI(f.URL)
```

Comment:
Looks like the code expects the URL to not have a trailing '/'.  Only question I have is if we input 'https://prow.ci.openshift.org/' does everything work or do we need to protect against that?   Looks like we are ok when I manually tested with 'https://prow.ci.openshift.org//view/gs/' so probably ok to ignore.

Reply:
Extra slashes are OK, if unsightly.